### PR TITLE
Fix macOS build

### PR DESCRIPTION
**Pull request by messense**
Also enabled macOS build on Travis CI to test it.

### CODE REVIEW

**Comment by siddontang** [comment]
Thanks @messense 

PTAL @BusyJay

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ cache:
 
 os:
   - linux
+  - osx
```
Comment (BusyJay) [inline .travis.yml]: Only stable compiler on OS X needs to be tested.

Comment (messense) [reply .travis.yml]: Done.

**Comment by messense** [comment]
@BusyJay PTAL

**Comment by BusyJay** [comment]
Thanks!

**Pull request merged**